src/v3/message.rs

```diff
@@ -0,0 +1,260 @@
+//! Provides types related to [`crate::v3::Message`].
```

Comment:
Sort of a nitpick but isn't this strictly the `MailSettings` type? Can we adjust the module name and some of the documentation here?

Reply:
I'm open to that. However, I didn't do that originally because this `MailSettings` type seemed specific to the `Message` schema. There's a separate [mail_settings](https://www.twilio.com/docs/sendgrid/api-reference/settings-mail/retrieve-all-mail-settings) API that returns a different structure that I didn't want this one to be confused with.

Just let me know if you'd still prefer to adjust the module name.

Reply:
Ah that's fair. Let's leave this for now then.